frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/reducers.ts

```diff
@@ -156,6 +173,13 @@ export default (state, action: WizardInternalAction) => {
         [dialogID, 'tabs', VMWizardTab.STORAGE, 'value'],
         payload.storageID,
       );
+    case InternalActionType.UpdateBus:
```

Comment:
IMO we should rather use SetStorages or UpdateStorage actions as this use case is too specific

Reply:
but once those are triggered the URL and container values getting cleared.

Reply:
we should update just the bus

Reply:
I tried it. but those actions erase the URL / container values.
so in each change in flavor/os/workload the values are erased. this action is useful to avoid that.

Reply:
Can you show me the code which does that? If there is some bug we should fix it then.

But it is not a good solution to create too specific actions like this one